pkg/asset/manifests/aws/cloudproviderconfig.go

```diff
@@ -0,0 +1,75 @@
+package aws
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/pkg/errors"
+	ini "gopkg.in/ini.v1"
+
+	awstypes "github.com/openshift/installer/pkg/types/aws"
+)
+
+// CloudConfig is the aws cloud provider config.
+type CloudConfig struct {
+	Global global
+}
+
+// global struct of CloudConfig which is currently not inialized.
+type global struct {
+	Zone                        string `ini:"Zone,omitempty"`
+	VPC                         string `ini:"VPC,omitempty"`
+	SubnetID                    string `ini:"SubnetID,omitempty"`
+	RouteTableID                string `ini:"RouteTableID,omitempty"`
+	RoleARN                     string `ini:"RoleARN,omitempty"`
+	KubernetesClusterTag        string `ini:"KubernetesClusterTag,omitempty"`
+	KubernetesClusterID         string `ini:"KubernetesClusterID,omitempty"`
+	DisableSecurityGroupIngress bool   `ini:"DisableSecurityGroupIngress,omitempty"`
+	ElbSecurityGroup            string `ini:"ElbSecurityGroup,omitempty"`
+	DisableStrictZoneCheck      bool   `ini:"DisableStrictZoneCheck,omitempty"`
+}
+
+// serviceOverride struct used for AWS service endpoint override.
+type serviceOverride struct {
+	Service       string `ini:"Service"`
+	Region        string `ini:"Region"`
+	URL           string `ini:"URL"`
+	SigningRegion string `ini:"SigningRegion,omitempty"`
```

Comment:
Similar to https://github.com/openshift/installer/pull/2073/files#r306990159, we should remove the fields beginning with Signing... we aren't using them.

Reply:
Removed unused fields from the struct.